README.md

```diff
@@ -1,2 +1,62 @@
+
 # resource-auditing
-Audit changes to K8s resources (e.g. NetworkPolicies)
+
+## Overview
+This project provides an auditing system for network policy-related, Kubernetes
```

Comment:
just re-read that, and it should be "Kubernetes resources and Antrea-specific CRDs"